examples/deployments/router_standalone_trtllm/worker.py

```
@@ -20,9 +20,11 @@
 
 DEFAULT_KV_EVENT_BUFFER_MAX_SIZE = 1024
 
+llm_max_num_tokens = int(os.getenv("TRTLLM_MAX_TOKENS", "8192")) 
 # Debug flag: set DYNAMO_DEBUG=1 to enable debug file dumps
 DEBUG_ENABLED = os.environ.get("DYNAMO_DEBUG", "0") == "1"
 DEBUG_WORKER_KV_FILE = "/tmp/debug_worker_kv.txt"
```

Ruff (0.14.10)

Probable insecure usage of temporary file or directory: "/tmp/debug_worker_kv.txt"

(S108)